arisia-remote/app/arisia/fun/DuckService.scala

```diff
@@ -99,7 +99,7 @@ class DuckServiceImpl(
           )
         } else {
           // Somehow got a request from the wrong place, which seems hinky:
-          val error = s"Got a request for Duck $duckId from incorrect location $from!"
+          val error = s"Got a request for Duck $duckId from incorrect location $from! It's supposed to be ${duck.requestingUrl}"
```

Comment:
We could also make the duck ids be 8 digit random strings so they can't be guessed as easily.

Reply:
:( nope backend restriction at the moment